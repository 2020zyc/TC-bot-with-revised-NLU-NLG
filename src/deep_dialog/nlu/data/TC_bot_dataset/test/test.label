inform
request+ticket
request+ticket
request+ticket
request+ticket
request+ticket
request+ticket
request+ticket
request+ticket
request+ticket
request+ticket
request+ticket
request+ticket
request+ticket
inform
inform
inform
inform
inform
inform
inform
inform
inform
inform
inform
inform
inform
inform
inform
inform
inform
inform+starttime
inform+starttime
thanks
request+date+starttime
request+date+starttime
request+starttime
request+starttime
request+starttime
request+starttime
request+starttime
request+starttime
request+starttime
request+starttime
request+starttime
request+theater+starttime
request+theater+starttime
request+theater+starttime
inform+starttime
inform+starttime
request+starttime
request+starttime
request+starttime
request+starttime
request+starttime
request+starttime
request+starttime
request+starttime
request+theater+starttime
request+theater+starttime
request+theater+starttime
request+theater
request+theater
request+theater
request+theater
request+theater
request+theater
request+theater
multiple_choice+theater
confirm_answer
request+ticket
request+ticket
request+ticket
inform
inform
inform
request+ticket
inform
inform
inform
inform
inform
inform
closing
inform
