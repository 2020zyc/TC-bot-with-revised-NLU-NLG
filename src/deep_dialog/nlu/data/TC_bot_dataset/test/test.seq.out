B-zip
O O O B-numberofpeople O
O O O B-numberofpeople O O B-moviename
O O O B-numberofpeople O
O O O O O O B-moviename
O O O O O O B-moviename I-moviename
O O O O O B-moviename I-moviename I-moviename B-date
O O O O O B-moviename I-moviename O B-theater I-theater I-theater
O O O O O B-moviename I-moviename B-date
O O O O O B-moviename B-date
O O O O O B-moviename O B-theater I-theater I-theater I-theater I-theater
O O O O O B-moviename B-date
O O O O O B-moviename O B-theater I-theater
O O O O O B-moviename B-date
O O B-numberofpeople O O
O O O O B-moviename I-moviename O B-starttime I-starttime
O O O O O O B-date I-date
O O O O O O B-date
O O O O O B-starttime I-starttime
O O O O O B-starttime I-starttime
O O O O O B-theater I-theater I-theater I-theater I-theater
O O O O O B-starttime I-starttime
O O O O O B-city
O O O O O B-theater I-theater I-theater
O O O O O B-starttime I-starttime I-starttime
O O O O O B-city
O O O O O B-starttime I-city
O O O O O B-date
O O O O O B-theater I-theater I-theater I-theater
O O O O B-moviename I-moviename I-moviename
O O B-numberofpeople O O
O O O B-numberofpeople O O O O O O O O O O O
O O O B-numberofpeople O O O O O O O O O O O
B-closing I-closing
O O O O O O O O B-city B-state
O O O O O O O O B-numberofpeople O O B-city I-city
O O O O O O B-moviename I-moviename I-moviename
O O O O O O B-moviename B-date
O O O O O O B-moviename I-moviename O B-theater I-theater
O O O O O O B-moviename I-moviename O B-city
O O O O O O B-moviename I-moviename B-date I-date
O O O O O O B-moviename O B-state
O O O O O O B-numberofpeople O O B-theater I-theater I-theater
O O O O O O B-numberofpeople O O B-moviename
O O O O O O B-numberofpeople O O B-city
O O O O O O O O B-state
O O O O O O O O O B-numberofpeople O
O O O O O O O O O B-numberofpeople O O B-city B-zip
O O O O O B-date I-date O O B-numberofpeople O
O O O O O B-date I-date O O B-numberofpeople O
O O O O O B-theater I-theater I-theater I-theater
O O O O O B-theater I-theater
O O O O O O B-date I-date
O O B-moviename I-moviename I-moviename O
O O B-moviename I-moviename I-moviename O
O O B-moviename O B-distanceconstraints I-distanceconstraints B-zip
O O B-moviename O O B-state
O O O O O B-city B-state
O O O O O O O O B-numberofpeople O O B-date
O O O O O O O O O B-numberofpeople O O B-city I-city
O O O O O O O O O B-numberofpeople O O B-city
O O O O O B-numberofpeople O O B-state
O O O O O B-numberofpeople O O B-moviename
O O O B-moviename O
O O O B-moviename I-moviename I-moviename I-moviename O
O O O O B-moviename I-moviename O B-city I-city
O O O B-city I-moviename O
O O O O O B-moviename O B-starttime I-starttime
O O O O
O
B-greeting O O O O O B-theater I-theater I-theater I-theater I-theater O
B-greeting O O O O O B-theater I-theater I-theater I-theater O
O O O O O O O O O O
O O O O O O
O O O O O O O
O O O B-numberofpeople O
O O O O O B-numberofpeople O
O O B-city I-city
O O B-city I-city
O O B-theater
O O B-moviename O O
B-date I-date
B-date
O
B-moviename
